misc/make-deps.sh

```diff
@@ -55,16 +59,23 @@ fi
 
 if [ $travis -eq 0 ]; then
 	if [ ! -z "$YUM" ]; then
+
+	    if [ -z "$GO" ]; then
```

Comment:
i think you've got a mix of tabs and spaces here and elsewhere. use only tabs please :)